metis-common/metis-common-base/src/main/java/eu/europeana/metis/common/SettingsHolder.java

```diff
@@ -0,0 +1,4 @@
+package eu.europeana.metis.common;
+
```

Comment:
A suggestion to take advantage of this interface:
I  see that `MongoIndexingSettings` has `MongoProperties` and `SolrIndexingSettings` has `SolrProperties`, perhaps you could create a `DatabaseProperties` interface which `MongoProperties` and `SolrProperties` implement
Then, this interface `SettingsHolder` could have a method called `getDatabaseProperties`, which as a consequence `MongoIndexingSettings` and `SolrIndexingSettings` will implement that method and return their specific properties 
This is just a suggestion of course, what do you think?